src/app/containers/MediaPageMain/index.jsx

```diff
@@ -45,18 +31,47 @@ const MediaPageMain = props => {
       <main role="main">
         <Grid>
           <GridItemConstrainedMedium>
-            {blocks.map(renderBlock({ script, service }))}
-            <ul>
-              <li>
-                <strong>Service</strong>: {service}
-              </li>
-              <li>
-                <strong>Brand</strong>: {serviceId}
-              </li>
-              <li>
-                <strong>MediaId</strong>: {mediaId}
-              </li>
-            </ul>
+            {blocks.map(({ id, text, type, live, externalId }) => {
+              const blockType = live ? LIVE_RADIO_BLOCK : type;
+
+              switch (blockType) {
+                case HEADING_BLOCK:
+                  return (
+                    <Headline
+                      key={id}
+                      script={script}
+                      service={service}
+                      id="content"
```

Comment:
Is the assumption there will only ever be one block of type "heading" safe?

We'll be duplicating IDs if not

Reply:
good point. I think that's there for the skip link. I can add a condition to only add the id when it's the first heading